### ISSUE TITLE
[Enhancement]: Allow download to mobile devices and/or through web page only to prevent leeching

### ISSUE THREAD

**Issue by ZLoth** [advplyr/audiobookshelf#2901]
### Describe the feature/enhancement

I know this is a bit of a unusual request, but for good reason. I want to be able to restrict the user so that they can download to their mobile device only and/or to their web browser.

As currently designed, if you enable downloading for a user, they can download from either the web browser or to their mobile device. However, the download consists of zipping up the files in a audiobook directory and sending it to the client's computer. This can pose some challenges as a friend could "leech" my entire audiobook collection. I want to be able to allow mobile downloads only, but stream only through the web browser. While this won't prevent leeching, it will be more difficult and time consuming. 

![image](https://github.com/advplyr/audiobookshelf/assets/6700159/d036510e-ec15-44ce-a4b5-ff1b1a3c00c8)

**Comment by nichwall** [comment]
There is nothing stopping them from downloading to a mobile device and sharing the files from there like you mentioned. Android is able to download files to shared storage by design and this feature is also planned for iOS.

There isn't really "extra effort or time" involved in this process other than tapping on an extra button when setting up the download location. If you're worried about people stealing files just don't give them download access. They could still stream from the server directly to a file if they are sufficiently motivated because ABS doesn't do any DRM.

**Comment by ZLoth** [comment]
> There is nothing stopping them from downloading to a mobile device and sharing the files from there like you mentioned. Android is able to download files to shared storage by design and this feature is also planned for iOS.

That's correct, and note that I didn't mention DRM. However, downloading the audiobook from the mobile device to the computer requires much more technical know-how and more effort to transfer to the computer than downloading from the web page. 

> There isn't really "extra effort or time" involved in this process other than tapping on an extra button when setting up the download location. If you're worried about people stealing files just don't give them download access. They could still stream from the server directly to a file if they are sufficiently motivated because ABS doesn't do any DRM.

Again, I don't want to block downloading to mobile devices because of data caps, I just want to make it harder to transfer to computer.
